test/integration/targets/aws_config/tasks/main.yaml

```diff
@@ -0,0 +1,459 @@
+---
+- block:
+
+    # ============================================================
+    # Prerequisites
+    # ============================================================
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          region: "{{ aws_region }}"
+      no_log: true
+
+    - name: ensure IAM role exists
+      iam_role:
+        <<: *aws_connection_info
+        name: AwsConfigRecorderTestRole
+        assume_role_policy_document: "{{ lookup('file','config-trust-policy.json') }}"
+        state: present
+        create_instance_profile: no
+        managed_policy:
+        - 'arn:aws:iam::aws:policy/service-role/AWSConfigRole'
+      register: config_iam_role
+
+    - name: ensure SNS topic exists
+      sns_topic:
+        <<: *aws_connection_info
+        name: AwsConfigDeliveryChannelTestTopic
+        state: present
+        subscriptions:
+          - endpoint: "rando_email_address@rando.com"
+            protocol: "email"
+      register: config_sns_topic
+
+    - name: ensure S3 bucket exists
+      s3_bucket:
+        <<: *aws_connection_info
+        name: "{{ config_s3_bucket | to_uuid }}"
```

Comment:
I think this will interfere if there are any parallel test runs, because both will get the same UUID from that seed string. The integration tests automatically populate a `resource_prefix` with the ID of the test run, so you could instead do: `{{ resource_prefix }}-config-s3-bucket` or similar.